app/src/main/res/values/strings.xml

```diff
@@ -6,7 +6,7 @@
     <string name="kde_connect" translatable="false">KDE Connect</string>
     <string name="title_shizuku" translatable="false">Shizuku</string>
 
-    <string name="label_wireless_debugging">Wireless debugging</string>
+    <string name="label_wireless_debugging">Wireless Debugging</string>
```

Comment:
In system settings, "wireless debugging" has no capital letters. Let's keep this the same here. The first letter here is capitalized because it is the first letter of the label.

![image](https://github.com/Smooth-E/wireless-adb-switch/assets/71522503/cc7a83d4-b319-4835-b09f-ea5968445d89)